docs/product-assembly_using-standalone-perspectives/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,30 @@
+[id='assembly_using-standalone-perspectives']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Using standalone perspectives
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: using-standalone-perspectives
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a developer, you can use the standalone perspectives from {CENTRAL} in your application.
+
+.Prerequisite
+* Ensure {CENTRAL} is deployed and is running in a web/application server.
```

Comment:
Should this be "Ensure {CENTRAL} is deployed and is running *on* a web/application server"

Reply:
Change acknowledged.